src/confirm-withdraw.ts

```diff
@@ -0,0 +1,106 @@
+import { Wallet, Provider } from "zksync-web3";
+import * as ethers from "ethers";
+import chalk from "chalk";
+import inquirer, { Answers, QuestionCollection } from "inquirer";
+import { track } from "./analytics";
+
+export default async function (
+  zeek?: boolean,
+  l1RpcUrl?: string,
+  l2RpcUrl?: string
+) {
+  track("confirm-withdrawal", { zeek, network: "goerli" });
```

Comment:
Did you want to keep this `track(...)`? It looks like you removed it at the beginning of the other commands